hack/lib/golang.sh

```diff
@@ -360,7 +360,7 @@ EOF
   local go_version
   IFS=" " read -ra go_version <<< "$(go version)"
   local minimum_go_version
-  minimum_go_version=go1.10.2
+  minimum_go_version=go1.11.1
```

Comment:
hm, lots of verify checks fail unless verify is bumped first, but bumping verify first means that any other PR will fail (due to gofmt failures).